uniffi/src/bindings/kotlin/templates/TopLevelFunctionTemplate.kt

```diff
@@ -6,12 +6,15 @@
             {{ arg.name() }}: {{ arg.type_()|type_kt }}{% if loop.last %}{% else %},{% endif %}
         {%- endfor %}
     ): {{ return_type|type_kt }} {
-        val _retval = _UniFFILib.INSTANCE.{{ func.ffi_func().name() }}(
-            {%- for arg in func.arguments() %}
-            {{ arg.name()|lower_kt(arg.type_()) }}{% if loop.last %}{% else %},{% endif %}
-            {%- endfor %}
-        )
-        return {{ "_retval"|lift_kt(return_type) }}
+        rustCall { e -> 
+            val _retval = _UniFFILib.INSTANCE.{{ func.ffi_func().name() }}(
+                {%- for arg in func.arguments() %}
+                {{ arg.name()|lower_kt(arg.type_()) }},
+                {%- endfor %}
+                e
+            )
+            return {{ "_retval"|lift_kt(return_type) }}
```

Comment:
I'll change it once I actually implement top level functions, but this return should be outside the rustCall probably